code/game/machinery/vending.dm

```diff
@@ -1010,7 +1010,8 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/head/cueball = 1,
 					/obj/item/clothing/under/scratch = 1,
         			/obj/item/clothing/under/sailor = 1,
-        			/obj/item/clothing/ears/headphones = 2)
+        			/obj/item/clothing/ears/headphones = 2,

```

Comment:
This indentation is offending me

Reply:
it's not staggered in the actual file; just preview.

Reply:
It's probably spaces instead of tabs, i'll normalize this.

Reply:
Better yet it's both.

Reply:
Find who did this and shoot them